content/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-personal-account-settings/permission-levels-for-a-personal-account-repository.md

```diff
@@ -53,6 +53,7 @@ The repository owner has full control of the repository. In addition to the acti
 | Create security advisories | "[About {% data variables.product.prodname_security_advisories %}](/github/managing-security-vulnerabilities/about-github-security-advisories)" |
 | Display a sponsor button | "[Displaying a sponsor button in your repository](/github/administering-a-repository/displaying-a-sponsor-button-in-your-repository)" |{% endif %}
 | Allow or disallow auto-merge for pull requests | "[Managing auto-merge for pull requests in your repository](/github/administering-a-repository/managing-auto-merge-for-pull-requests-in-your-repository)" | 
+| Manage webhooks and deploy keys	| "[Managing deploy keys](/developers/overview/managing-deploy-keys#deploy-keys)" |
```

Comment:
Hi @jsoref - I read the notes in [your issue](https://github.com/github/docs/issues/16788) and you make a good point about a missing link to information about webhooks in the original content.

I think it would be okay to split this into two rows and use either of the following links for webhooks:

- `[About webhooks](/developers/webhooks-and-events/webhooks/about-webhooks)`
- `[Webhooks](/developers/webhooks-and-events/webhooks)`

Reply:
Neither are great, since they're really about how one handles a hook as opposed to a thing that talks about adding/deleting/disabling/otherwise managing webhooks from the github side.

https://docs.github.com/en/developers/webhooks-and-events/webhooks/creating-webhooks#setting-up-a-webhook sort-of covers adding (It says "click the thing", but then you'll be prompted w/ lots of choices -- for which a page would come in real handy...) -- it doesn't cover:

> Which events would you like to trigger this webhook?
> Just the push event.
> Send me everything.
> Let me select individual events.

And the article we're reading is in the context of doing a specific thing, not actually setting up an arbitrary webhook.

The bottom of:
https://docs.github.com/en/rest/webhooks#subscribing
has a link for the various things for repository webhooks, but there isn't even an anchor for that, and it doesn't really work well...

Fwiw, I actually *have* webhooks, so I know there's a very complicated ui for modifying an existing webhook, or disabling it (or trying to -- ime, I can't disable it, but that's really a different bug).

I think I'd rather leave that detail unresolved -- I could file a request asking for a proper page, but I'd expect it'd have to be tossed into the internal side anyway since it's entirely new content and at best I can hand-wave at what it should cover.

Reply:
Thanks for taking the time to explain what's currently missing (as you'll have guessed I haven't used webhooks directly).

I'll take your feedback and raise an internal issue for this, I think that you're right that this needs some careful thought.